0000-template.md

```diff
@@ -0,0 +1,26 @@
+# <Feature Name>
+
+## Summary
+[summary]: #summary
+
+One paragraph explanation of the feature.
+
+## Motivation
+[motivation]: #motivation
+
+Why are we doing this? What use cases does it support? What is the expected outcome?
+
+## Approach
+[approach]: #approach
+
+Provide a brief description of the implementation details for the feature. What
+existing code paths are you planning to modify. What new classes or
+datastructures are you planning to introduce.
+
+## Questions
```

Comment:
How about calling it as `Discussion Points` instead of `Questions`? 
I think this section is to discuss among contributors to find the best solution instead of "asking" cartographer team.

Reply:
Done